logpyle/__init__.py

```diff
@@ -975,6 +976,9 @@ def __call__(self, lst):
                 elif agg_name == "avg":
                     from pytools import average
```

Comment:
maybe this could be changed to `statistics.fmean`?

Reply:
:+1: